### ISSUE TITLE
Debian 12 support

### ISSUE THREAD

**Issue by de4ps** [GoogleCloudPlatform/ops-agent#1294]
**Description of the Issue**

On Debian 12, the installation of the `google-cloud-ops-agent` package is failing. The exact error message being received is:

```
E: Unable to locate package google-cloud-ops-agent
[2023-06-20T14:42:22+0000] google-cloud-ops-agent installation failed.
```

**Steps to Reproduce**

1. On a Debian 12 system, attempt to install the `google-cloud-ops-agent` package via terminal.

**Expected Behavior**

The `google-cloud-ops-agent` package should install successfully without any error messages.

**Actual Behavior**

The installation fails, throwing an error message that the package cannot be located.

**Environment**

- OS: Debian 12
- Repository: `https://packages.cloud.google.com/apt google-cloud-ops-agent-bookworm-all InRelease`

Any help resolving this issue would be greatly appreciated.

**Comment by braydonk** [comment]
Hi @de4ps thank you for opening an issue. The team is currently actively working Debian 12 support, we are hoping to have it ready ASAP.

**Comment by braydonk** [comment]
Unfortunately we don't have an ETA right now, but we will keep this issue updated when it is ready.

**Comment by xanadonf** [comment]
I've also been tricked... and I'm wondering: why offer VM installations in debian 12 when few things are compatible with it.
Anyway.
Let's hope the patch doesn't take 8 months like with debian 11.

**Comment by ridwanmsharif** [comment]
Debian 12 support is available in Ops Agent v2.35.0: https://github.com/GoogleCloudPlatform/ops-agent/releases/tag/2.35.0

Installing the latest agent release by following https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/installation should do the trick.
